core/src/main/java/org/mskcc/cbio/portal/model/GeneticProfile.java

```diff
@@ -32,14 +32,15 @@
 
 package org.mskcc.cbio.portal.model;
 
+import java.io.Serializable;
 import java.util.Properties;
 
 import org.apache.commons.lang.builder.ToStringBuilder;
 
 /**
  * Class for genetic profile
  */
-public class GeneticProfile {
+public class GeneticProfile implements Serializable {
```

Comment:
A member of this class is of type GeneticAlterationType ... but that is an enum and inherently serializable (so no declaration needed)